docs/databags.md

```diff
@@ -33,6 +33,14 @@ The License Hash is part of a data bag item encrypted with [Chef Vault](https://
 
 * `[A Decriptive File-Name]` - Corresponding License (XML) contents. (There can be many of these) Remember to change newlines to `\n` to conform to proper JSON format.
 
+License Pool Hash
+------------
+This hash is part of a plaintext data bag item that defines multiple license pools. Indexers can be assigned to a specific pool and license quota can be set for each pool by configuring this databag. Quota can be specified in integer values of KB, MB, GB or TB and must include the unit (e.g. '400GB' or '400GiB).
+
+* `['auto_generated_pool_size']` - This is the pool size for the auto generated pool. Indexers that connect to the license server, when not assinged to a specific pool will land in this pool.
+* `['pools'][pool_name]['max_size']` - Maximum size for the pool
```

Comment:
I think the attribute name was confusing. I thought of it as the maximum quota assigned to a pool.
To avoid the confusion, I updated it to _size_ 
d2dee5b5317bea995913c8453324f8c0a6fffd25